e2e/cluster/local/scripts/cluster-up

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
nit: maybe a `DEBUG` variable can be useful here to give more visibility with -x for example